githooks/apps/runner/runner.go

```diff
@@ -602,6 +602,11 @@ func checkSharedHook(
 	allAddedHooks *[]string,
 	sharedType hooks.SharedHookType) bool {
 
+	// Aborting a 'reference-transaction' hook (type 'prepared') leads
+	// to all sorts of problems, therefore
+	// do only print an error and continue.
+	isFatal := settings.HookName == "reference-transaction"
```

Comment:
Based on the comment above this, shouldn't everything else be fatal but not this?